leap/leap.go

```diff
@@ -7,17 +7,11 @@ func main() {
 	fmt.Print("Enter the year: ")
 	var year int
 	fmt.Scan(&year)
-	if year%100 == 0 {
-		if year%400 == 0 {
-			fmt.Println(year, "is a leap year.")
-		} else {
-			fmt.Println(year, "is a not leap year.")
-		}
+	if year%100 == 0 && year%400 == 0 {
```

Comment:
Just for fun, please try to combine century and non-century conditions to have that `fmt.Println(year, "is not a leap year")` only once in the code